### ISSUE TITLE
Mixer Shortcuts

### ISSUE THREAD

**Issue by SH4DOWSIX** [musescore/MuseScore#20092]
### Your idea

Ability to mute/solo tracks on the mixer through shortcuts.

I use musescore for practicing along with the track and often mute the track and keep the metronome going during playback.
I was previously using a plugin on MS 3 that did the same thing.

### Problem to be solved

I use musescore for practicing along with the track and often mute the track and keep the metronome going during playback.

### Prior art

I was previously using a plugin on MS 3 that did the same thing.

**Comment by jrbowden** [comment]
I agree, a shortcut key to mute or to solo the current track when in the main score window would be a great feature. Most DAWs and sequencers have this kind of function, without the need to go into the mixer first.
Great suggestion.I'd also like a shortcut key to toggle the metronome as well.

**Comment by MarcSabatella** [comment]
Note you can already solo instruments without the mixer - just make a range selection.  So if you currently have a single note selected where you want to start playback, press Shift+Right to create a small range selection, and now Space will solo that instrument.  This also works for soloing multiple staves, as long as they are contiguous so you can range-select them.

But additional controls could be nice as wel.

**Comment by jmwmusic** [comment]
A number of blind users would really like to see these shortcuts implemented.  Navigating back to the mixer/playback panel with tabs and arrows is slow and some of them aren't even very accessible.  It also means hearing a lot of screen reader speech in between which makes music comparison hard for arrangers/composers comparing two things or working with aural skills to hear bits on just 1 staff that they have written.

Being able to press a shortcut to Solo the current staff or mute the current staff would be very helpful.  Being able to pick 2 or more non adjacent staves to solo would work with this.

The suggestion on the MuseScoreAccessibility group today was for Shift Alt M to mute the current staff, Shift Alt S to solo the current staff and Shift Alt R to restore all to default, ie off.

There has also been a request to be able to solo just the right or left hands of a piano staff #20053. Necessary for blind piano players learning mainly by ear and those arranging and composing for piano.  Could this be incorporated so that pressing these shortcuts mutes/solos the current piano staff, not the whole piano part?

**Comment by stevage** [comment]
To spell out this feature a bit more, since I'd also like it:

A shortcut-able action "mute instrument":

- whichever instruments are selected in the Layout panel are affected. (Range or list selections of notes will automatically select the relevant parts in the Layout panel, but you can also manually select parts).
- if any of those instruments are currently not muted, then all the affected instruments become muted
- otherwise (if all those instruments are currently muted), then all the affected instruments become unmuted

And similar for "solo instrument".

This would also partially address #24878 since you could quickly select a bar across the section you're interested in and use this shortcut.

Ideally, it would provide some immediate feedback about whether you just muted or unmuted, but currently there doesn't seem to be any real way to do that.
